webrtc.html

```diff
@@ -2087,19 +2126,7 @@
               </li>
 
               <li>
-                <p>If <var>connection</var>'s <a href=
-                "#dom-peerconnection-signaling-state"><code>RTCPeerConnection</code>
-                signalingState</a> is <code>stable</code>, then fire an <a href=
-                "#event-negotiation">negotiationneeded</a> event at
-                <var>connection</var>.</p>
-
-                <div class="note">
-                  <p>Throughout this document, we need to indicate what should
-                  happen if a change happens and the state is not stable, most
-                  likely to fire the negotiationneeded event when the state
-                  eventually returns to stable.
```

Comment:
I think this needs to be added in two places only:
- SetLocalDescription's "set signalingState accordingly" (step 7 of the last list in setLocalDescription): If resulting state is "stable", and "needs_to_fire_negotiation_needed" flag is set, fire NN.
- SetRemoteDescription's "set signalingState accordingly" (this text is missing in the current draft).

Question: Should the needs_to_fire_negotiation_needed flag be an observable property?

(I've invented the term "needs_to_fire_negotiation_needed" as "what gets set when "Mark connection as needing negotiation" is done).